### PR TITLE
feat: Pagination size change select component is searchable

### LINKED ISSUE #27684: Pagination component increases the number of records per page that can be input.Now you can only make a drop-down selection.
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
I can show the records I want on one page

### What does the proposed API look like?
To improve the showsizechanger parameter, you can input and drop down.

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/27684

### 💡 Background and solution

图片

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Pagination size change select component is searchable now.         |
| 🇨🇳 Chinese |  Pagination 的页码选择器可以进行搜索了。          |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 7cf9dbe</samp>

Added `showSearch` prop to `MiniSelect` and `MiddleSelect` components in `components/pagination/Select.tsx` to enable search input in pagination select dropdowns. This improves the user experience and accessibility of the pagination component.

### <samp>🤖 Generated by Copilot at 7cf9dbe</samp>

* Add `showSearch` prop to `MiniSelect` and `MiddleSelect` components to enable search input in select dropdown ([link](https://github.com/ant-design/ant-design/pull/42608/files?diff=unified&w=0#diff-a34e1dd1a02f6ee6067986a1242ba2b093bb074eafa1739c050af8d2dd1edc52L9-R10))

### CODE REVIEW

File: components/config-provider/__tests__/__snapshots__/components.test.tsx.snap [hunk 1]
```diff
@@ -19007,11 +19007,8 @@ exports[`ConfigProvider components Pagination configProvider 1`] = `
               autocomplete="off"
               class="config-select-selection-search-input"
               id="rc_select_TEST_OR_SSR"
-              readonly=""
```
Comment (kiner-tang) [inline components/config-provider/__tests__/__snapshots__/components.test.tsx.snap:19010]: 这些属性没了符合预期么

**Review by li-jia-nan — APPROVED** [review]

**Review by kiner-tang — APPROVED** [review]

File: components/config-provider/__tests__/__snapshots__/components.test.tsx.snap (hunk 1 above)
Comment (afc163) [reply components/config-provider/__tests__/__snapshots__/components.test.tsx.snap:19010]: 符合预期，input 可以搜索就不是 readonly 了。

**Comment by chenzhihang950909** [comment]
@afc163 请问能不能给个配置 把这个下拉框的搜索功能关掉。因为我们不需要这个搜索功能。

**Comment by afc163** [comment]
@chenzhihang950909 有这个功能会有什么负面影响么？

**Comment by chenzhihang950909** [comment]
@afc163 因为，我们的只提供了4个选择，20 30 40 50 的大小，如果再多个搜索功能，显得很累赘。

**Comment by afc163** [comment]
多个搜索，貌似也不会影响用户选择？

**Comment by chenzhihang950909** [comment]
害，主要是，产品，认为有点多余。- -

**Comment by afc163** [comment]
有理有据去沟通就好了，我判断不会对用户交互有明显的额外影响。

**Pull request merged**